.gitignore

```diff
@@ -5,3 +5,4 @@ typedoc/
 dist/
 lib/
 .DS_Store
+.idea
```

Comment:
this can go into your `$HOME/.gitignore_global`

Reply:
Good tip. I haven't use the global ignore file before. I reverted the change to the `.gitignore`